rule_schema_v1.yaml

```diff
@@ -482,13 +514,11 @@ $defs:
         title: Inspect a metavariable with a given analyzer
         properties:
           analyzer:
-            type: string
-            title: Analyzer to use
+            $ref: "#/$defs/analyzer"
           metavariable:
             type: string
             title: Metavariable to analyze
-          options:
-            type: object
+          additionalProperties: false
```

Comment:
do we want that? Will this create problems down the line when we update the schema and add new fields?
or we can just rely on min-max version to skip the rule?

Reply:
First, I made a mistake here. `additionalProperties:` should not be at the same level as `metavariable`.

Regardless, it's a good question. `additionalProperties` protects the user against spelling mistakes but comes with compatibility issues. I think I'll remove it.

Note that we could check additional fields for spelling mistakes and report an error if they're too close to a missing optional field name. We'd need to be careful to report e.g. `massage` but not `message2` if an optional `message` field isn't specified.